src/commands/utility/info/channel.js

```diff
@@ -0,0 +1,36 @@
+const { Command, SwitchbladeEmbed } = require('../../../')
+const moment = require('moment')
+
+module.exports = class InfoChannel extends Command {
+  constructor (client) {
+    super({
+      name: 'channel',
+      parent: 'info',
+      requirements: { guildOnly: true },
+      parameters: [{
+        type: 'channel',
+        required: true,
+        acceptText: true,
+        acceptNews: true,
+        acceptStore: true,
+        onlySameGuild: true,
+        missingError: 'commands:move.missingChannelMention'
+      }]
+    }, client)
+  }
+
+  async run ({ t, author, channel, language }, givenChannel) {
+    moment.locale(language)
+    const embed = new SwitchbladeEmbed(author)
+
+      .setTitle(`${givenChannel.nsfw ? `${t('commands:info.subcommands.channel.nsfwTag')} ` : ''}${givenChannel.name}`)
+      .setDescription(
+        `${givenChannel.parent ? `**${t('commands:info.subcommands.channel.category')}**: ${givenChannel.parent.name}` : ''}
+        **${t('commands:guildinfo.createdAt')}**: ${moment(givenChannel.createdAt).format('LLL')}\n(${moment(givenChannel.createdAt).fromNow()})
+        **ID**: ${givenChannel.id} <#${givenChannel.id}>
+        **${t(givenChannel.members.size > 1 ? t('commands:info.subcommands.channel.canViewThisChannelPlural', { userCount: givenChannel.members.size }) : t('commands:info.subcommands.channel.canViewThisChannel', { userCount: givenChannel.members.size }))}**
```

Comment:
Maybe putting these informations on a separated variable for better reading

Reply:
Example

Reply:
Well, take the last line for example:
```js
        **${t(givenChannel.members.size > 1 ? t('commands:info.subcommands.channel.canViewThisChannelPlural', { userCount: givenChannel.members.size }) : t('commands:info.subcommands.channel.canViewThisChannel', { userCount: givenChannel.members.size }))}**
```

this line is very big, you could turn this into a variable, like:

```js
	const canViewText = t(`commands:info.subcommands.channel.canViewThisChannel${givenChannel.members.size > 1 ? 'Plural : ''}`, userCount: givenChannel.members.size)
	...
		**${canView}**
```

btw, why is there a `t` inside another `t`?
```
${t(givenChannel.members.size > 1 ? t('...', { ... }) : ...
  ^                                 ^ 
```